pkg/capabilities/cli/cmd/template_workflow_generator_helper.go

```diff
@@ -0,0 +1,71 @@
+package cmd
+
+import (
+	"bytes"
+	_ "embed"
+	"strings"
+	"text/template"
+
+	"github.com/iancoleman/strcase"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+)
+
+//go:embed templates/go_workflow_builder.go.tmpl
+var goWorkflowTemplate string
+
+type TemplateWorkflowGeneratorHelper struct {
+	Templates map[string]string
+}
+
+func (t *TemplateWorkflowGeneratorHelper) Generate(info GeneratedInfo) (map[string]string, error) {
+	files := map[string]string{}
+	if t.Templates == nil {
+		return files, nil
+	}
+
+	for file, t := range t.Templates {
+		content, err := genFromTemplate(file, t, info)
+		if err != nil {
+			return nil, err
+		}
+
+		// can use a template, but it's simple for now
+		fileName, err := genFromTemplate("file name for "+file, file, info)
```

Comment:
this is bizarre. genFromTemplate is both creating content and creating file names?

Reply:
genFromTemplate just runs template.  I'm using a template to get the file name.